packages/sdk/src/credential/teamsUserCredential.browser.ts

```diff
@@ -219,73 +254,23 @@ export class TeamsUserCredential implements TokenCredential {
     return getUserInfoFromSsoToken(ssoToken.token);
   }
 
-  private async exchangeAccessTokenFromSimpleAuthServer(
-    scopesStr: string,
-    authCodeResult: AuthCodeResult
-  ): Promise<void> {
-    const axiosInstance: AxiosInstance = await this.getAxiosInstance();
-
-    let retryCount = 0;
-    while (true) {
-      try {
-        const response = await axiosInstance.post("/auth/token", {
-          scope: scopesStr,
-          code: authCodeResult.code,
-          code_verifier: authCodeResult.codeVerifier,
-          redirect_uri: authCodeResult.redirectUri,
-          grant_type: GrantType.authCode,
-        });
-
-        const tokenResult: AccessTokenResult = response.data;
-        const key = await this.getAccessTokenCacheKey(scopesStr);
-        // Important: tokens are stored in sessionStorage, read more here: https://aka.ms/teamsfx-session-storage-notice
-        this.setTokenCache(key, {
-          token: tokenResult.access_token,
-          expiresOnTimestamp: tokenResult.expires_on,
-        });
-        return;
-      } catch (err: any) {
-        if (err.response?.data?.type && err.response.data.type === "AadUiRequiredException") {
-          internalLogger.warn("Exchange access token failed, retry...");
-          if (retryCount < maxRetryCount) {
-            await this.sleep(retryTimeSpanInMillisecond);
-            retryCount++;
-            continue;
-          }
-        }
-        throw err;
-      }
-    }
-  }
-
-  /**
-   * Get access token cache from authentication server
-   * @returns Access token
-   */
-  private async getAndCacheAccessTokenFromSimpleAuthServer(
-    scopesStr: string
-  ): Promise<AccessToken> {
-    try {
-      internalLogger.verbose(
-        "Get access token from authentication server with scopes: " + scopesStr
-      );
-      const axiosInstance: AxiosInstance = await this.getAxiosInstance();
-      const response = await axiosInstance.post("/auth/token", {
-        scope: scopesStr,
-        grant_type: GrantType.ssoToken,
-      });
+  private async init(): Promise<void> {
+    microsoftTeams.initialize();
```

Comment:
Looks like the `initialize()` is duplicated with the one in getSSOToken. I think it can be moved here so each getSSOToken invocation won't check timeout issues once initialized.

Reply:
Remove this line this teams sdk will ONLY be called in getSSOToken during initialization.